spring-learning/src/main/java/spring/learning/video/three/Triangle.java

```diff
@@ -1,39 +1,23 @@
 package spring.learning.video.three;
 
-public class Triangle {
-
-    private Point pointA;
-    private Point pointB;
-    private Point pointC;
+import java.util.List;
 
-    public Point getPointA() {
-        return pointA;
-    }
-
-    public void setPointA(Point pointA) {
-        this.pointA = pointA;
-    }
-
-    public Point getPointB() {
-        return pointB;
-    }
+public class Triangle {
 
-    public void setPointB(Point pointB) {
-        this.pointB = pointB;
+    public List<Point> getPoints() {
+        return points;
     }
 
-    public Point getPointC() {
-        return pointC;
+    public void setPoints(List<Point> points) {
+        this.points = points;
     }
 
-    public void setPointC(Point pointC) {
-        this.pointC = pointC;
-    }
+    private List<Point> points;
 
     public void draw() {
```

Comment:
Please rearange the fields.

The attributes should be first.
Then the actual methods.
Then the getters and setters.